test/performance/scenario2.py

```diff
@@ -0,0 +1,50 @@
+import json
```

Comment:
What is the difference between the application and test folder, they look like the same files to me. If this is the case we should not have code duplication but have a common call